frontend/src/metabase/home/reducers.js

```diff
@@ -15,3 +19,10 @@ export const recentViews = handleActions(
   },
   [],
 );
+
+export const lastSeenCollection = handleActions(
```

Comment:
Not sure if this would be the best place to put it - I was thinking of putting it in the `app` part of the store but I felt that since that we're dealing with the home page for this logic, it should go here